src/lib.rs

```diff
@@ -62,7 +62,13 @@ pub async fn init_world_tree(
 
     if config.cache.purge_cache {
         tracing::info!("Purging tree cache");
-        fs::remove_file(&config.cache.cache_file)?;
+        if config.cache.cache_file.exists() {
+            fs::remove_file(&config.cache.cache_file)?;
+        }
+
+        // There's something wrong with CascadingMerkleTree impl
```

Comment:
Is this still the case?

Reply:
Happy to set create(true) if we want to.

Reply:
Yeah, I think it's still the case